src/Studio/Plugins/Devices/BciDeviceWidget.cpp

```diff
@@ -135,12 +129,10 @@ void BciDeviceWidget::AddSensorItems(QTreeWidgetItem* parent)
 		// add signal quality subitem
 		item = new QTreeWidgetItem(sensorItem);
 		item->setText(0, "Signal Quality");
-		item->setHidden(NEUROMORE_MINIMAL_DEVICE_STATS);
```

Comment:
Theres so much you've changed that I'm dubious about. Please resubmit the PR again.

Reply:
@dioptre 
Please check my new PR [New_PR(#102)](https://github.com/neuromore/studio/pull/102). I have changed a few things according to your issues in this PR.